nhost/compose/config.go

```diff
@@ -171,6 +171,27 @@ func (c Config) postgresConnectionString() string {
 	return fmt.Sprintf("postgres://%s:%s@%s:%d/%s", user, password, SvcPostgres, dbPort, db)
 }
 
+func (c Config) hasuraConnectionString() string {
```

Comment:
I see 3 identical functions that have only different usernames. Make a single one which takes an a single `username`argument

```
func (c Config) postgresConnectionStringWithUsername(username string) string {
	postgresEnv := c.postgresServiceEnvs()
	db := postgresEnv[envPostgresDb]

	return fmt.Sprintf("postgres://%s@%s:%d/%s", username,  SvcPostgres, dbPort, db)
}
```